test/nodes/test_vectordb_qdrant.py

```
@@ -0,0 +1,351 @@
+        ig.validateConfig()
+        assert any('invalid' in m.lower() for m in warned_messages)
+
+    def test_port_zero_warns(self, mock_config, warned_messages):
+        """Port 0 should produce a warning."""
+        config = {'host': 'localhost', 'port': 0, 'apikey': '', 'collection': 'test-col'}
+        ig = self._make_iglobal(config, mock_config)
+        ig.validateConfig()
+        assert any('Port' in m for m in warned_messages)
+
+    @pytest.mark.parametrize(
+        'host,port,expected_scheme',
```

Ruff (0.15.7)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)